controllers/updateservice_controller_test.go

```diff
@@ -642,7 +642,7 @@ func newRequest(updateservice *cv1.UpdateService) reconcile.Request {
 }
 
 func newTestReconciler(initObjs ...runtime.Object) *UpdateServiceReconciler {
-	c := fake.NewFakeClient(initObjs...)
+	c := fake.NewClientBuilder().WithRuntimeObjects(initObjs...).WithStatusSubresource(&cv1.UpdateService{}).Build()
```

Comment:
This is a good one.

https://github.com/kubernetes-sigs/controller-runtime/issues/2362#issuecomment-1837270195